specification/components/r4/schemas/responses/retrieveBinary/307Response.yaml

```diff
@@ -0,0 +1,12 @@
+description: Response redirects user agent to the large file attachment URL in the Object Store via the Location header.
```

Comment:
We shouldn't be referencing "large file attachments" externally really, perhaps this description could be something like ".... the user agent to a location within the Object Store from which the Binary can be downloaded from via the Location header".